backends/dpdk/DpdkXfail.cmake

```diff
@@ -65,3 +65,18 @@ p4c_add_xfail_reason("dpdk"
   "Unhandled declaration type"
   testdata/p4_16_samples/psa-test.p4
   )
+
+p4c_add_xfail_reason("dpdk"
+  "Only one LPM match field is permitted per table"
+  testdata/p4_16_samples/psa-dpdk-lpm-match-err1.p4
+  )
+
+p4c_add_xfail_reason("dpdk"
+  "Non 'exact' match kind not permitted"
+  testdata/p4_16_samples/psa-dpdk-lpm-match-err2.p4
+  )
+
+p4c_add_xfail_reason("dpdk"
+  "Key field wider than 64-bit is not permitted"
+  testdata/p4_16_samples/psa-dpdk-lpm-match-err3.p4
+  )
```

Comment:
If the message "key field wider than 64-bit is not permitted" is an accurate description of current DPDK back end restrictions, does that mean that if someone wants to do a match on an IPv6 address, they must break it up into 2 separate 64-bit key fields, e.g. hdr.ipv6.dstAddress[127:64] and hdr.ipv6.dstAddress[63:0] ?

Reply:
Yes. Currently, we are only checking this and giving an error. But we do plan to remove this restriction soon by splitting such fields into fields with width within 64-bit.